src/task 22/task22.js

```diff
@@ -0,0 +1,30 @@
+function buildTable(arr){
+    var Table = document.querySelector("#table");
+    var headers = Object.keys(arr[0]);
```

Comment:
Still the same problem appears. What if your data set is not full with data? Specifically, if 0th element? As
```
var MOUNTAINS = [
  {name: "Kilimanjaro", country: "Tanzania"},
  {name: "Everest", height: 8848, country: "Nepal"},
  {name: "Mount Fuji", height: 3776, country: "Japan"},
  {height: 4808, country: "Italy/France"},
  {name: "Vaalserberg", height: 323, country: "Netherlands"},
  {name: "Denali", height: 6168, country: "United States"},
  {name: "Popocatepetl", height: 5465}
];
```
You will miss the whole column. Think about it.